src/main/Main.hs

```diff
@@ -786,6 +792,7 @@ execCmd ExecOpts {..} go@GlobalOpts{..} =
         ExecOptsPlain -> do
           (cmd, args) <- case (eoCmd, eoArgs) of
                 (ExecCmd cmd, args) -> return (cmd, args)
+                (ExecRun, args) -> return ("", args)
```

Comment:
Can you explain why we need an empty command case here? Is this a code path which shouldn't be triggered? If so, I would rather generate an error message here, since the `exec` call below will fail anyway.